src/app.js

```diff
@@ -2,4 +2,163 @@ const express = require('express');
 
 const app = express();
 
+app.use(express.json());
+
+const {
+  sayHello,
+  uppercase,
+  lowercase,
+  firstCharacter,
+  firstCharacters,
+} = require('./lib/strings');
+
+const { add, multiply, subtract, divide, remainder } = require('./lib/numbers');
+
+const {
+  getNthElement,
+  arrayToCSVString,
+  addToArray,
+  elementsStartingWithAVowel,
+  removeNthElement2,
+} = require('./lib/arrays');
+
+const { negate, truthiness, isOdd, startsWith } = require('./lib/booleans');
+
+app.get('/strings/hello/:string', (req, res) => {
+  const { string } = req.params;
+  res.json({ result: sayHello(string) });
+});
+
+app.get('/strings/upper/:string', (req, res) => {
+  const { string } = req.params;
+  res.json({ result: uppercase(string) });
+});
+
+app.get('/strings/lower/:string', (req, res) => {
+  res.json({ result: lowercase(req.params.string) });
+});
+
+app.get('/strings/first-characters/:string', (req, res) => {
+  if (!req.query.length) {
+    res.json({ result: firstCharacter(req.params.string) });
+  } else {
+    res.json({ result: firstCharacters(req.params.string, req.query.length) });
+  }
+});
+
+app.get('/numbers/add/:num1/and/:num2', (req, res) => {
+  const num1 = parseInt(req.params.num1, 10);
+  const num2 = parseInt(req.params.num2, 10);
+  if (Number.isNaN(num1) || Number.isNaN(num2)) {
+    res.status(400).json({ error: 'Parameters must be valid numbers.' });
+  } else {
+    res.json({ result: add(num1, num2) });
+  }
+});
+
+app.get('/numbers/subtract/:num1/from/:num2', (req, res) => {
+  const num1 = parseInt(req.params.num1, 10);
+  const num2 = parseInt(req.params.num2, 10);
+  if (Number.isNaN(num1) || Number.isNaN(num2)) {
+    res.status(400).json({ error: 'Parameters must be valid numbers.' });
+  } else {
+    res.status(200).json({ result: subtract(num2, num1) });
+  }
+});
+
+app.post('/numbers/multiply/', (req, res) => {
+  const { a, b } = req.body;
+  if (a === undefined || b === undefined) {
```

Comment:
Same as above, instead of if/if else/ else
you could have 2 if and a statement at the end if both if's are not relevant:
````
  const { a, b } = req.body;
  if (a === undefined || b === undefined) {
    res.status(400).json({ error: 'Parameters "a" and "b" are required.' });
  }
  if (Number.isNaN(Number(a)) || Number.isNaN(Number(b))) {
    res.status(400).json({ error: 'Parameters "a" and "b" must be valid numbers.' });
  }
  res.status(200).json({ result: multiply(a, b) });
`````
But this is all just formatting really for better readability